storage/remote/otlptranslator/prometheusremotewrite/metrics_to_prw_test.go

```diff
@@ -90,14 +91,9 @@ func TestFromMetrics(t *testing.T) {
 					}
 
 					seenFamilyNames[wantMetric.familyName] = struct{}{}
-					expMetadata = append(expMetadata, prompb.MetricMetadata{
-						Type:             wantMetric.metricType,
-						MetricFamilyName: wantMetric.familyName,
-						Help:             wantMetric.description,
-						Unit:             wantMetric.unit,
-					})
 				}
 
+				// TODO check returned counters
```

Comment:
Not sure what you mean @dashpole  , there are more explicit detailed test further down to test the counters, gauges, histograms regarding type and value.

Reply:
Don't remember adding that TODO :).  You can remove it if you've determined it isn't neccessary.